### PR TITLE
Misc deployer fixes: 

### PR DESCRIPTION
**Pull request by banaag**
1. Change repo for docker build to amppackager1 (per gibbley@) to fix main repo triggering
2. Update Makefile to use REGISTRY for SOURCE_REGISTRY
3. Update schema.yaml wrong default values and use persistence storage size in yaml.

### CODE REVIEW

File: k8s/amppackager/Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ APP_ID ?= $(CHART_NAME)
 
 TRACK ?= 1.0
 
-SOURCE_REGISTRY ?= marketplace.gcr.io/google
+SOURCE_REGISTRY ?= REGISTRY 
 IMAGE_AMPPACKAGER ?= $(SOURCE_REGISTRY)/amppackager:$(TRACK)
```
Comment (gibbleyg) [inline k8s/amppackager/Makefile:12]: `amppackager1`

File: k8s/amppackager/Makefile [hunk 2]
```diff
@@ -8,7 +8,7 @@ APP_ID ?= $(CHART_NAME)
...
 IMAGE_INIT_CONTAINER ?= $(SOURCE_REGISTRY)/amppackager/init:$(TRACK)
```
Comment (gibbleyg) [inline k8s/amppackager/Makefile:13]:
```suggestion
IMAGE_AMPPACKAGER ?= $(SOURCE_REGISTRY)/amppackager1:$(TRACK)
IMAGE_INIT_CONTAINER ?= $(SOURCE_REGISTRY)/amppackager1/init:$(TRACK)
```

Comment (banaag) [reply k8s/amppackager/Makefile]: Done.

**Review by gibbleyg — APPROVED** [review]

File: k8s/amppackager/schema.yaml [hunk 3]
```diff
@@ -105,7 +105,7 @@ properties:
     type: string
     title: Domain
     description: The filename for the private key used by amppackager.
-    default: "amppkg.privkey"
+    default: "your-domain-url.com"
```
Comment (twifkak) [inline k8s/amppackager/schema.yaml:108]: Surprising to me - doesn't look like a typical filename.

**Review by twifkak — APPROVED** [review]

**Pull request merged**